study-builder/tenants/rgp/emails/change_password.html

```diff
@@ -0,0 +1,127 @@
+{% assign p3 = "Reset my password" %}
+{% assign p4 = "Sincerely," %}
+{% assign p5 = "The Rare Genomes Project Team" %}
+{% assign p6 = "If you did not make this request, please contact us." %}
+{% endif %}
+
+<center>
+    <table style="background: #ffffff; width: 600px; margin-top: 50px; padding: 0; -webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%; mso-table-lspace: 0pt; mso-table-rspace: 0pt; font-family: Roboto, sans-serif; border-collapse: collapse !important;"
+           align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+        <tr style="width: 600px;">
+            <td align="left" style="margin: 0; padding: 40px 0 40px 50px; font-family: Roboto, sans-serif; width: 175px;">
+                <img src="https://image.ibb.co/hSFqi5/logo.jpg" alt="Rare Genomes Project logo" style="-ms-interpolation-mode: bicubic;border: 0;height: auto; outline: none;text-decoration: none;">
+            </td>
+            <td align="right" style="margin: 0; padding: 40px 50px 40px 0; font-family: Roboto, sans-serif; width: 425px;">
+                <h1 style="font-family: Roboto, sans-serif; font-weight: 600; color: #f55714; font-size: 19px;">{{ heading }}</h1>
+            </td>
+        </tr>
+    </table>
+
+    <table style="background: #ffffff; width: 600px; margin: 0; padding: 0; -webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%; mso-table-lspace: 0pt; mso-table-rspace: 0pt; font-family: Roboto, sans-serif; border-collapse: collapse !important;" align="center"
+           border="0" cellpadding="0" cellspacing="0" width="100%">
+        <tr style="width: 600px;">
+            <td style="padding: 0 50px 40px 50px;">
+                <p style="font-family: 'Roboto', sans-serif; text-align: left; font-weight: 300; font-size: 16px;">{{ p1 }}</p>
+                <p style="font-family: 'Roboto', sans-serif; text-align: left; font-weight: 300; font-size: 16px;">{{ p2 }}</p>
+
+                <p style="padding: 25px 0 20px 0; text-align: center;">
+                    <a href="{{ url }}" style="background-color: #f55714 !important; color: #ffffff !important; font-family: 'Roboto', sans-serif; border-radius: 23px; font-weight: 500; font-size: 18px; padding: 13px 25px 12px 25px; cursor: pointer; text-align: center; text-decoration: none;">{{ p3 }}</a>
+                </p>
+
+                <p style="font-family: 'Roboto', sans-serif; text-align: left; font-weight: 300; font-size: 16px;">{{ p4 }}</p>
+                <p style="font-family: 'Roboto', sans-serif; text-align: left; font-weight: 500; padding-top: 10px; color: #0C457D; font-size: 16px;">{{ p5 }}</p>
+
+                <hr style="border: 2px solid #EAEEF3; border-bottom: 0; margin: 20px 0;">
+                <p style="font-family: 'Roboto', sans-serif; text-align: center; color: #A9B3BC;-webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%;font-size: 13px;">
+                   {{$p6}}
```

Comment:
I think this is the wrong syntax?

Reply:
Not sure how I missed this one!  I'm going to take a closer look when I test this time and compare side-by-side...